modules/client/examples/03-multisig-client/05-can-approve.ts

```diff
@@ -0,0 +1,35 @@
+/* MARKDOWN
+### Check if user can approve a transaction (Multisig)
+
+Checks whether a user is able to participate in a proposal created using the Multisig plugin.
+*/
+
+import {
+  CanApproveParams,
+  ContextPlugin,
+  MultisigClient
+} from "@aragon/sdk-client";
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiate a plugin context from the aragonOSx SDK context.
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Instantiate a Multisig client.
+const client = new MultisigClient(contextPlugin);
+
+const canApproveParams: CanApproveParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890",
+  addressOrEns: "0x1234567890123456789012345678901234567890",
+  proposalId: BigInt(0)
+};
+
+// Checks whether the signer of the transaction can approve a given proposal.
```

Comment:
Judging by the parameters described above, I think this is out of sync. The given `addressOfEns` should be the one that can approve or not?

@josemarinas ?

Reply:
@josemarinas 👀